### DISCUSSION TITLE
git

### DISCUSSION THREAD

**Discussion by F-one-1** [vuepress/core#507] [Q&A] [answered]
I performed the following operations as an example

```
echo '.temp' >> .gitignore
echo '.cache' >> .gitignore
```
Now I want to remove it from `.gitignore`

What should I do？

**Comment by Mister-Hope** [comment, marked as answer]
Please use vim. 

### Also, dicussion panel is a place to discuss VuePress related problems, please find answers of **basic questions** like this in places like stackoverflow.
